src/styles/partials/mixin.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 @mixin fontLarge {
-  font-size: 1.5rem;
+  font-size: 3rem;
   line-height: 2rem;
   font-weight: 700;
```

Comment:
There is no need in this a its been previously set up in global.scss. As for style guide we only using font weight 400 and 600